### DISCUSSION TITLE
0.50.0 feedback

### DISCUSSION THREAD

**Discussion by SuricateTrading** [imaNNeo/fl_chart#1027] [General]
I just updated to newest version and it's really a pain in the §$% to update 50% of my chart code just because of a single feature to support widget titles for like 1% of the people who actually need this feature.
I think there could have been better solutions which wouldn't break so much.

**Comment by imaNNeo** [comment]
I know what you are talking about. 

Let me mention my point of view:

1. When we talk about the importance, we can sort issues by thumbs-up count.
#183 is the 9th issue when you sort all the issues with thumbs-up count.
Then I think it is more than 1% of people.

2. We are in the `0.x.x` version range (we are not `1.0.0` yet). One of the most important reasons that I don't bump our version up to 1.0.0 is that I feel at this time we cannot guarantee that there will be no breaking changes. (It's something like beta version)

3. Generally I understand the problem and sorry about that. I'll do my best to prevent this kind of sever breaking changes,
Sometimes It needs a lot of work to prevent breaking changes which I cannot do much work.

It the end, thanks a lot for your feedback. We couldn't develop this library without your valuable feedbacks.

**Reply by mars000** [reply]
quick simple question - how do I create margins nows in 0.50.x example: I have sideTitle name and I want to create some space between it and the axislabels. Previoulsy I used margin: for this.

**Reply by imaNNeo** [reply]
Oh sorry. I wrote a migration guide when I released it. But it seems I forgot to mention it.
Check the migration guide [here](https://github.com/imaNNeoFighT/fl_chart/blob/master/repo_files/documentations/migration_guides/0.50.0/MIGRATION_00_50_00.md).

**Reply by large** [reply]
I get a 404 on this migration guide

**Reply by imaNNeo** [reply]
Sorry, I just updated the link
